Backend/app/dashboard/Lib/src/API/Dashboard/Person.hs

```diff
@@ -44,13 +44,18 @@ type API =
              :> Post '[JSON] APISuccess
            :<|> DashboardAuth 'DASHBOARD_ADMIN
              :> Capture "personId" (Id DP.Person)
-             :> "assignMerchantAccess"
-             :> ReqBody '[JSON] DPerson.MerchantAccessReq
+             :> "assignMerchantCityAccess"
```

Comment:
Backward compatibility issues may come. We have to do it properly.

Reply:
Do not remove existing API. Add new API and in existing APIs use default city.

Reply:
Fixed. Please check.